#include "team_ms_player.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

namespace msplayerservice{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

///<<<rpc begin
void PlayerPlayerTeamServiceImpl::EnterScene(common::EntityPtr& entity,
    const msplayerservice::TeamTestRequest* request,
    msplayerservice::TeamTestResponse* response)
{
///<<< BEGIN WRITING YOUR CODE EnterScene
///<<< END WRITING YOUR CODE EnterScene
}

///<<<rpc end
}// namespace msplayerservice
